core/src/main/java/org/infinispan/xsite/irac/DefaultIracManager.java

```diff
@@ -557,23 +577,33 @@ private void onBatchResponse(IracBatchSendResult result, Collection<? extends Ir
       if (log.isTraceEnabled()) {
          log.tracef("[IRAC] Batch completed with %d keys applied. Global result=%s", successfulSent.size(), result);
       }
+      Collection<IracManagerKeyState> doneKeys = new ArrayList<>(successfulSent.size());
       switch (result) {
          case OK:
-            iracExecutor.disableBackOff();
+            backOffHandler.resetBackOff();
```

Comment:
this is not working. 
```
2023-05-22 18:59:34,491 ERROR (blocking-thread--p3-t3) [o.j.p.r.RELAY2] JGRP000290: pedro-desktop-14560:site1: no route to site site3: dropping message (89 identical messages for site site3 in the last 60629 ms)
2023-05-22 19:00:34,646 ERROR (blocking-thread--p3-t3) [o.j.p.r.RELAY2] JGRP000290: pedro-desktop-14560:site1: no route to site site3: dropping message (85 identical messages for site site3 in the last 60155 ms)
2023-05-22 19:01:35,197 ERROR (blocking-thread--p3-t3) [o.j.p.r.RELAY2] JGRP000290: pedro-desktop-14560:site1: no route to site site3: dropping message (86 identical messages for site site3 in the last 60551 ms)
```

I think each site has to have its own `org.infinispan.util.ExponentialBackOff` in `org.infinispan.xsite.irac.IracXSiteBackup`, so the `reset` only resets the timer for that site.

Reply:
Updated, now each site has its own backoff timing.